src/features/cart/screen/index.tsx

```diff
@@ -75,10 +75,10 @@ const CartScreen = () => {
         value={search}
         containerStyle={styles.searchBarContainer}
       />
-      {products && products.data.length !== 0 && search.length > 0 ? (
+      {products && products.length !== 0 && search.length > 0 ? (
```

Comment:
The `length` will be evaluated to a truthy value, most of the times there is no need to check if is greater to zero.
```suggestion
      {products.length && search.length ? (
```